environment.yml

```
@@ -0,0 +1,14 @@
+name: donor-data-pipeline
+dependencies:
+- python=3
+- cython=0.29.13
+- pandas=0.25.0
+- requests=2.22.0
+- pylint=2.3.1
+- pip=19.2.2
+- pytest=5.0.1
+- pytest-cov=2.7.1
+- sphinx=2.1.2
+
```

Adding a different theme for Sphinx generated docs
```suggestion
sphinx-readable-theme=1.3.0

```